x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/host_details.tsx

```diff
@@ -93,13 +93,40 @@ export const HostDetails = memo(({ details }: { details: HostMetadata }) => {
 
   const policyStatusClickHandler = useNavigateByRouterEventHandler(policyResponseRoutePath);
 
+  const [policyDetailsUri, policyDetailsRoutePath] = useMemo(() => {
```

Comment:
are the names switched here?
`*Path` I normally associate with the router path
`*Uri` or `*Url` I normally associated with an Href type of prop.

Reply:
They were, corrected them - my mistake